config/application.rb

```diff
@@ -32,6 +33,16 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
+    # Active Job Configurations
+    redis_config = Scholarsphere::RedisConfig.new
+
+    # config.active_job.queue_name_prefix = "scholarsphere_production"
```

Comment:
@whereismyjetpack is the queue name still relevant? If not, I'd remove this line. Otherwise, we could do something like `config.active_job.queue_name_prefix = "scholarsphere_#{Rails.env}"`

Reply:
I'm not sure that it is. we can always add it later if we decide it's needed.